clients/flash/air-client/src/org/bigbluebutton/air/chat/views/TopToolbarMediatorChat.as

```diff
@@ -23,10 +20,6 @@ package org.bigbluebutton.air.chat.views {
 			}
 		}
 		
-		override protected function leftButtonClickHandler(e:MouseEvent):void {
-			uiSession.popPage();

```

Comment:
It has been removed because it was executing the wrong thing. Now `leftButtonClickHandler` exectues what is implemented in the parent class `TopToolbarMediatorAIR`, it just inherits it.